packages/camera/camera/ios/Classes/CameraPlugin.m

```diff
@@ -10,6 +10,7 @@
 #import <CoreMotion/CoreMotion.h>
 #import <libkern/OSAtomic.h>
 #import <uuid/uuid.h>
+#import "FLTFlashMode.h"
```

Comment:
I think `FlashMode.h` as you had it was better, there's no reason to prefix header names (that I'm aware of), even though a few other files have done so.

Reply:
You're right, that is the pattern, and Apple also prefixes their header names.  Hard to keep it all straight. 🙂